test/BUILD

```diff
@@ -38,6 +38,13 @@ plz_e2e_test(
     expected_output = "query_somepath_nopath_test.txt",
 )
 
+plz_e2e_test(
+    name = "query_somepath_hidden_test",
+    cmd = "HIDDEN=`plz query somepath //tools/build_langserver:build_langserver //tools/build_langserver/lsp:all --hidden | wc -l`; \
```

Comment:
super nit: The back tick stuff is somewhat discouraged now days. We should really be using the `$(plz query ...)` syntax instead.